### ISSUE TITLE
"Successful" blob saving for iPhone15,2 but saves no blobs

### ISSUE THREAD

**Issue by UInt2048** [airsquared/blobsaver#476]
**Describe the bug**
Expected behavior: Trying to save blobs with APNonce and generator grabbed from device works
Actual behavior: Registers as success but nothing is added to blob folder or to TSSSaver folders (TSSSaver URL is [here](https://stor.tsssaver.1conan.com/tsssaver/shsh/1135916207521822/)), and fails on SHSH.host

**Screenshots**
![image](https://user-images.githubusercontent.com/46784000/190902088-745ab6b2-c8c5-405c-8c81-4cff2b677d28.png)

**Desktop:**
 - OS Version: macOS 12.6
 - iOS Device: iPhone 14 Pro (iPhone15,2) on iOS 16.0.1

**Checklist (place an x between the brackets to mark as completed):**
 - [x] I have made sure I am on the latest version of blobsaver
 - [x] I have checked other open issues to see if my problem has already been reported
 - [x] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
iosVersions = [IOSVersion[versionString=16.0.1, ipswURL=https://updates.cdn-apple.com/2022FallFCS/fullrestores/012-69357/7F866BFA-FCEF-4A01-8597-B345A29F0559/iPhone15,2_16.0.1_20A371_Restore.ipsw, signed=true], IOSVersion[versionString=16.0, ipswURL=https://updates.cdn-apple.com/2022FallFCS/fullrestores/012-66000/78F42B45-7562-4808-83A8-FDF43C4B5FAC/iPhone15,2_16.0_20A362_Restore.ipsw, signed=true]]
Directly downloaded to /var/folders/w7/4_hxk65j4z56mvz8qxfpl2hh0000gn/T/BuildManifest6699228076573667902.plist
Running: [/Applications/blobsaver.app/Contents/MacOS/tsschecker, --nocache, --save, --device, iPhone15,2, --ecid, 4091C1A0B401E, --save-path, /Users/matthewbenedict/Blobs, --boardconfig, D73AP, --apnonce, b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c, --generator, 0xb9eba0abbf282205, --build-manifest, /private/var/folders/w7/4_hxk65j4z56mvz8qxfpl2hh0000gn/T/BuildManifest6699228076573667902.plist]
tsschecker version: 0.390-0002d69ca4793ceac9d35e45a9323e7788e2d902
libfragmentzip version: 0.1-aaf6fae83a0aa6f7aae1c94721857076d04a14e8-RELEASE
[TSSC] manually specified generator "0xb9eba0abbf282205"
[TSSC] manually specified ECID to use, parsed "4091C1A0B401E" to dec:1135916207521822 hex:4091c1a0b401e
[TSSC] manually specified apnonce to use, parsed "b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c" to hex:b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c
[TSSC] opening /private/var/folders/w7/4_hxk65j4z56mvz8qxfpl2hh0000gn/T/BuildManifest6699228076573667902.plist
[TSSR] Checking BuildIdentity 0
[TSSR] Selected BuildIdentity for request
[TSSR] LOG: device iPhone15,2 doesn't need a baseband ticket, continuing without requesting a Baseband ticket
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... failure
[Error] ERROR: TSS request failed (status=69, message=This device isn't eligible for the requested build.)

iOS 16.0.1 for device iPhone15,2 IS NOT being signed!
Warning: ignoring unsigned version; API is likely out of date
Directly downloaded to /var/folders/w7/4_hxk65j4z56mvz8qxfpl2hh0000gn/T/BuildManifest2775826999911304518.plist
Running: [/Applications/blobsaver.app/Contents/MacOS/tsschecker, --nocache, --save, --device, iPhone15,2, --ecid, 4091C1A0B401E, --save-path, /Users/matthewbenedict/Blobs, --boardconfig, D73AP, --apnonce, b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c, --generator, 0xb9eba0abbf282205, --build-manifest, /private/var/folders/w7/4_hxk65j4z56mvz8qxfpl2hh0000gn/T/BuildManifest2775826999911304518.plist]
tsschecker version: 0.390-0002d69ca4793ceac9d35e45a9323e7788e2d902
libfragmentzip version: 0.1-aaf6fae83a0aa6f7aae1c94721857076d04a14e8-RELEASE
[TSSC] manually specified generator "0xb9eba0abbf282205"
[TSSC] manually specified ECID to use, parsed "4091C1A0B401E" to dec:1135916207521822 hex:4091c1a0b401e
[TSSC] manually specified apnonce to use, parsed "b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c" to hex:b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c
[TSSC] opening /private/var/folders/w7/4_hxk65j4z56mvz8qxfpl2hh0000gn/T/BuildManifest2775826999911304518.plist
[TSSR] Checking BuildIdentity 0
[TSSR] Selected BuildIdentity for request
[TSSR] LOG: device iPhone15,2 doesn't need a baseband ticket, continuing without requesting a Baseband ticket
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... failure
[Error] ERROR: TSS request failed (status=69, message=This device isn't eligible for the requested build.)

iOS 16.0 for device iPhone15,2 IS NOT being signed!
Warning: ignoring unsigned version; API is likely out of date
{deviceIdentifier=iPhone15,2, boardConfig=D73AP, ecid=1135916207521822, generator=0xb9eba0abbf282205, apnonce=b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c}
{
    "ecid": "1135916207521822",
    "url": "https://tsssaver.1conan.com/shsh/1135916207521822",
    "jobs": 3
}
{selected_firmware=All, ecid=4091C1A0B401E, generator=0xb9eba0abbf282205, device=iPhone15,2, apnonce=b81d392f059ddf0769c19352b67a3f25db1c27781f98b371df54be17ad4f743c, boardconfig=D73AP}
{
    "code": 902,
    "message": "20A362 and 20A371 failed"
}

```

**Comment by lawleagle** [comment]
+1, I get the same behaviour

**Comment by jcerda0001** [comment]
Ditto. I get the blobs successfully saved but there’s nothing inside the Blobs folder. Any fix for this?

**Comment by TheEight8all** [comment]
I can't even get the blobs I keep getting an error. I might be doing something wrong. Can you walk us through how you did it please?

**Comment by UInt2048** [comment]
@TheEight8all,

1. Put the firmware identifier (e.g. iPhone15,2) and board config (e.g. D73AP) in the respective places. You can obtain this by searching for your device on The iPhone Wiki.
2. Read ECID from device
3. Read APNonce & generator from device
4. Check also save to TSSSaver & shsh.host
5. Try to save blobs

**Comment by UInt2048** [comment]
I have successfully saved blobs elsewhere thanks to the help in DanTheMann15/tsschecker#28.

Posting this in case the devs of Blobsaver would like to use this fork as a backend.

**Comment by mido-777** [comment]
@UInt2048 
Thanks I failed to mention I'm on the iPhone 14 pro max and my Mac has to be updated to newer OS to even read my phone contents. I'll try that and follow your steps. Thanks again

**Comment by UInt2048** [comment]
I believe this is still an issue

**Comment by TheEight8all** [comment]
Yea it is, still haven't been to fix even after going through the device support update

**Comment by airsquared** [comment]
This has been fixed in v3.3.0.
